internal/cephfs/volume.go

```diff
@@ -88,7 +88,7 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 		if errors.Is(err, rados.ErrNotFound) {
 			return nil, ErrVolumeNotFound
 		}
-		// In case the error is other than invalid command return error to the caller.
+		// Incase the error is other than invalid command return error to the caller.
```

Comment:
Should this not be two words? I thought you added similar changes for this with with the new spellcheck GitHub CI job...

Reply:
Yes got messed up. corrected now